docs/tutorials/web.md

```diff
@@ -5,10 +5,405 @@ has_children: false
+
+    // 2. Loop through the image buffer and extract the R, G, and B channels
+    for (let i = 0; i < imageBufferData.length; i += 4) {
+      redArray.push(imageBufferData[i]);
+      greenArray.push(imageBufferData[i + 1]);
+      blueArray.push(imageBufferData[i + 2]);
+      // skip data[i + 3] to filter out the alpha channel
+    }
+
+    // 3. Concatenate RGB to transpose [224, 224, 3] -> [3, 224, 224] to a number array
+    const transposedData = redArray.concat(greenArray).concat(blueArray);
+
+    // 4. convert to float32
+    let i, l = transposedData.length; // length, we need this for the loop
+    // create the Float32Array size 3 * 224 * 224 for these dimensions output
+    const float32Data = new Float32Array(dims[1] * dims[2] * dims[3]);
+    for (i = 0; i < l; i++) {
+      float32Data[i] = transposedData[i] / 255.0; // convert to float
+    }
+    // 5. create the tensor object from onnxruntime-web.
+      const inputTensor = new Tensor("float32", float32Data, dims);
+      return inputTensor;
+  }
+
+```
+### modelHelper.ts
+
+The inputTensor is ready for inferencing. Let's call the default `modelHelper.ts` function and walk through the logic. First we create the `ort.InferenceSession` by sending in the path to the model and the `SessionOptions`. For the `executionProviders` you can use either `webgl` to use the GPU or `wasm` to use the CPU. See the documentation to learn more about the `SessionOptions` available for inferencing configuration [here](https://onnxruntime.ai/docs/api/js/interfaces/InferenceSession.SessionOptions.html).
+
+```javascript
+import * as ort from 'onnxruntime-web';
+import _ from 'lodash';
+import {imagenetClasses} from '../data/imagenet';
+import { InferenceSession } from 'onnxruntime-web';
+
+const [predictions, inferenceTime] = await runSqueezenetModel(imageTensor);
+
+export async function runSqueezenetModel(preprocessedData: any): Promise<[any, number]> {
+    
+    // Create session and set options. See the docs here for more options: 
+    //https://onnxruntime.ai/docs/api/js/interfaces/InferenceSession.SessionOptions.html#graphOptimizationLevel
+
+    const session = await ort.InferenceSession
+                            .create('./_next/static/chunks/pages/squeezenet1_1.onnx', 
+                            { executionProviders: ['webgl'], graphOptimizationLevel: 'all' });
+    console.log('Inference session created')
+
+    // Run inference and get results.
+    var [results, inferenceTime] =  await runInference(session, preprocessedData);
+    return [results, inferenceTime];
+ 
+  }
+
+```
+
+Then let's call the `runInference` function by sending in the `session` and our input tensor `preprocessedData`.
+
+```javascript
+  async function runInference(session: InferenceSession, preprocessedData: any): Promise<[any, number]> {
+
+    // Get start time to calculate inference time.
+    const start = new Date();
```

Comment:
use `Performance.now()` instead of `Date`

Reply:
In this scenario I want to use Date.Now now Performance.Now to measure time.